db/migrate/20170421215941_create_submission_packets.rb

```diff
@@ -0,0 +1,9 @@
+class CreateSubmissionPackets < ActiveRecord::Migration[5.0]
+  def change
+    create_table :submission_packets do |t|
+      t.references :poem, foreign_key: true, index: true, null: false
+      t.references :submission, foreign_key: true, index: true, null: false
+      t.string :status
```

Comment:
I'm not sure at this point. The status of an individual `SubmissionPacket` only changes when the parent `Poem` or `Submission` does, but there's no meaningful initial state. I'll make a placeholder initial status to avoid the having an empty column, though.